pkg/sql/crdb_internal.go

```diff
@@ -1116,7 +1116,7 @@ func populateSystemJobsTableRows(
 		getLegacyPayload := func(ctx context.Context) (*jobspb.Payload, error) {
 			return payload, nil
 		}
-		err = jobsauth.Authorize(
+		err = jobsauth.AuthorizeAllowLegacyAuth(
```

Comment:
I think we this change could use a test that runs each of the cross cluster pg functions with an unauthorized user to ensure we don't slip in a change that removes the authorization check.

Reply:
yeah, thats a reasonable idea. i also plan to metamorphically run all PCR and LDR tests not from root.